apps/studio/src/components/AvatarMenu.tsx

```diff
@@ -0,0 +1,35 @@
+import { Divider, Flex, Text } from "@chakra-ui/react"
+import {
+  AvatarMenu as AvatarMenuComponent,
+  Menu,
+} from "@opengovsg/design-system-react"
+import { BiUser } from "react-icons/bi"
+
+import { useMe } from "~/features/me/api"
+
+export const AvatarMenu = () => {
+  const { me, logout } = useMe()
+
+  return (
+    <AvatarMenuComponent
+      name={me.name}
+      variant="subtle"
+      bg="base.canvas.brand-subtle"
+      menuListProps={{ maxWidth: "19rem" }}
+    >
+      <Menu.Item
+        isDisabled
+        style={{ cursor: "default", backgroundColor: "transparent" }}
+      >
+        <Flex alignItems="center" gap="0.75rem">
+          <BiUser size="1.25rem" color="#666c7a" />
+          <Text textStyle="subhead-1" textColor="base.content.medium">
+            {me.email}
+          </Text>
+        </Flex>
+      </Menu.Item>
+      <Divider orientation="horizontal" />
+      <Menu.Item onClick={() => logout()}>Sign out</Menu.Item>
```

Comment:
suggestion(design): will leave to design to comment on this one but was thinking that since we have a logo for the email, I thought would be good to have a logout logo here as well, so that the text can be aligned. It would look something like this:

![image](https://github.com/user-attachments/assets/df62ec59-9aef-4ac3-9fc6-ca9b16838ec3)

Reply:
good suggestion! will check with Sehyun

Reply:
added in https://github.com/opengovsg/isomer/pull/999/commits/60805b2fd7ca3280df91cd3ac86c72ca8d7ae7e0